### PR TITLE
Added private chat support + minor improvements

### PR DESCRIPTION
**Pull request by DerDodo**

### CODE REVIEW

**Review by PaulBFB — APPROVED** [review]
thank you for changing the variables from camelCase <3

**Review by PaulBFB — APPROVED** [review]
nice code structure improvements 👍

File: src/data_structures.py [hunk 1]
```diff
@@ -19,8 +20,19 @@ class SlackMessage:
     replies: list[SlackThreadMessage]
 
 
+class ChannelType(Enum):
```
Comment (guilhermeprokisch) [inline src/data_structures.py:23]: If you already using python 3.11 it's could be a  good idea to use strEnum here.

File: src/SlackDumpReader.py [hunk 2]
```diff
@@ -2,16 +2,23 @@
...
 
 class SlackDumpReader:
+    data_cleaner: SlackDataCleaner
+
+    def __init__(self, data_cleaner: SlackDataCleaner):
+        self.data_cleaner = data_cleaner
+
     def read(self, file_path: str) -> SlackData:
         dump_file = open(file_path, encoding="utf-8")
+        file_name = os.path.basename(file_path).split("/")[-1][:-5]
```
Comment (guilhermeprokisch) [inline src/SlackDumpReader.py:21]: In this cases I prefer to use pathlib. It's more readable I guess

```python
from pathlib import Path

file_name = Path(file_path).name 
```

**Review by guilhermeprokisch — APPROVED** [review]

**Pull request merged**